### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by cig0** [sunny0826/kubecm#18]
Hi,

Me again :)

Please consider expanding `kubecm` with additional functionality specially useful when using aliases:

```
  kubecm {context,c} <NAME>  (to easily switch to context <NAME>)
  kubecm {context,c} -       (to quickly switch to the previous context)
  kubecm {context,c}         (to quickly find out the current context name. Optionally maybe a flag like --current could be useful)
```

% kubecm version
Version: 0.13.0
GoOs: darwin
GoArch: amd64

Thanks! 🙏

**Comment by sunny0826** [comment]
This function looks very similar to `switch`, have you tried the `switch` subcommand?

`switch` allows you to switch contexts quickly and has a search function.

**Comment by chenyg0911** [comment]
Yes, I think if put the context as a parameter, It would help to use alias or call it from a shell script.

**Comment by cig0** [comment]
@sunny0826 sure, but in its current form you are able to select a context interactively, whereas something like `kubecm s _context_` would be a very welcome addition :)

**Comment by sunny0826** [comment]
Thank you for the suggestion, I will add this feature

**Comment by sunny0826** [comment]
This feature has been released in [v0.13.1](https://github.com/sunny0826/kubecm/releases/tag/v0.13.1)

you can use command 

```shell
kubecm s <NAME>
```
